config/server.yml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: strest-server
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: strest-server
+  template:
+    metadata:
+      labels:
+        app: strest-server
+    spec:
+      containers:
+      - name: strest-server
+        image: buoyantio/strest-grpc:0.0.7
```

Comment:
consider `latest` ?  or, update the `Releasing` section in the README to bump the version here.